pkg/aws/client.go

```diff
@@ -741,7 +745,11 @@ func (c *awsClient) GetAWSAccessKeys() (*AccessKey, error) {
 }
 
 func (c *awsClient) GetLocalAWSAccessKeys() (*AccessKey, error) {
-	creds, err := c.awsSession.Config.Credentials.Get()
+	cfg, err := config.LoadDefaultConfig(context.TODO())
```

Comment:
Do we have `cfg` stored in the `awsClient` struct? or do we need to retrieve it?